models/iris/iris_scanner_definition.yaml

```diff
@@ -0,0 +1,94 @@
+# The `scan` section contains general information for the scan job.
+#scan:
+#  output:
+#    path: './reports'  # location where reports are written - if absent will default to './reports'
+
+
+
+# The `model_use_case` section describes general information about the problem being solved by the ML models in this scan.
+# Multiple scan definitions may be associated with a single model_use_case.
+model_use_case:
+  model_use_case_id: c12e/datasciencelab/iris
+  author: 'info@cognitivescale.com'
+  name: 'My Purpose: Value Prediction'
+  task_type: multiclass-classification
+  performance_metrics:
+  - name: 'Accuracy'
+    metric: 'Accuracy'
+  atx_performance_metric_name: 'Accuracy'
+
+# The `models` section provides the details for each of the models being evaluated as part of the use case.
+models:
+- model_id: 'svm'
+  name: 'Support Vector Machine'
+  description: 'A simple SVM model'
+  predict_endpoint: http://127.0.0.1:8551/predict
+
+# The `datasets` section lists the datasets that may be used to run this evaluation.
+datasets:
+- dataset_id: eval
+  url: ./iris_eval.csv
+  has_header: true
+  file_type: csv
+  delimiter: ','
+  quote_character: '"'
+
+
+# The `dataset_schema` section contains schema details about the datasets to be evaluated.
+dataset_schema:
+  outcome_column: 'species'
+  feature_schemas:
+  - feature_name: sepal_length
+    data_type: numerical-float
+    min: 4.3
+    max: 7.9
+    spread: 0.7000000000000002
```

Comment:
for numerical features, this is not necessarily needed as it is computed but doesn't do any harm to include